server/lib/report_server/reports/student_actions.ex

```diff
@@ -0,0 +1,88 @@
+defmodule ReportServer.Reports.StudentActions do
+  use ReportServer.Reports.Report, type: :athena
+
+  alias ReportServer.PortalDbs
+
+  def get_query(report_filter = %ReportFilter{}, user = %User{}) do
+    run_remote_endpoints = get_run_remote_endpoints(report_filter, user)
+      |> Enum.map(&("'#{&1}'"))
+      |> Enum.join(", ")
+
+    # FIXME: when we add non-admin access update this code to only allow admins to not hide names
+    hide_names = report_filter.hide_names
+    remove_username = false
```

Comment:
What is the difference between these two options?

Reply:
That is in there for future work - I ported the code from Javascript and it is used in the joined learner reports.